plugins/pipelines/app/views/samson_pipelines/_fields.html.erb

```diff
@@ -0,0 +1,23 @@
+<fieldset>
+  <legend>Pipelining</legend>
+  <p class="col-lg-offset-2">Set the stage that will get automatically deployed after this one.</p>
```

Comment:
This doesn't quite line up because its missing the [grid-gutter](https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss#L327), can you make this also a `col-lg-10` or whatever?